.github/workflows/ci.yml

```diff
@@ -4,7 +4,8 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
+    branches:
+    - '**'
```

Comment:
I think you just want to get rid of these lines (looking at some other files).
```suggestion
```

Reply:
No dice :/